extern/sector-storage/storiface/resources.go

```diff
@@ -1,19 +1,31 @@
-package sectorstorage
+package storiface
 
 import (
-	"github.com/filecoin-project/go-state-types/abi"
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"golang.org/x/xerrors"
 
+	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/lotus/extern/sector-storage/sealtasks"
 )
 
 type Resources struct {
-	MinMemory uint64 // What Must be in RAM for decent perf
-	MaxMemory uint64 // Memory required (swap + ram)
+	MinMemory uint64 `envname:"MIN_MEMORY"` // What Must be in RAM for decent perf
+	MaxMemory uint64 `envname:"MAX_MEMORY"` // Memory required (swap + ram)
```

Comment:
"What Must be in RAM for decent perf" makes sense to me, but "Memory required (swap + ram)" is confusing me a bit given that the name is MAX.  It sounds like this is MIN_MEMORY + MIN_SWAP?  I think clarifying this comment would be helpful.